infrastructure/src/main/java/org/corfudb/infrastructure/NettyServerRouter.java

```diff
@@ -31,8 +33,39 @@
     public static final String PREFIX_EPOCH = "SERVER_EPOCH";
     public static final String KEY_EPOCH = "CURRENT";
 
-    ExecutorService handlerWorkers = Executors.newWorkStealingPool(Runtime.getRuntime()
-            .availableProcessors());
+    public static class ServerThreadFactory
+            implements ForkJoinPool.ForkJoinWorkerThreadFactory {
+
+        final String threadPrefix = "ServerRouter-";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this final field static too. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1170)